sdk/iot/Azure.ResourceManager.IotFirmwareDefense/src/Customization/Models/CryptoCertificateResult.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.IotFirmwareDefense.Models
+{
+    /// <summary> Crypto certificate resource. </summary>
+    public partial class CryptoCertificateResult : ResourceData
+    {
+        /// <summary> Name of the certificate. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string NamePropertiesName
+        {
+            get => Name;
+            set{ return; }
+        }
```

Comment:
I think this is not correct.
If we have readd the flattening - I think we no longer need these customizations unless their property names have been changed.
If we did not add back the flattening, there should be a `Properties` property carrying these properties, the implementation should be redirecting to that property instead like this:
```
public string NamePropertiesName
{
	get => Properties?.Name;
	set
	{
		Properties ??= new();
		Properties.Name = value;
	}
}
```
so that when people are using this `NamePropertiesName`, it will behave exactly the same as if they are getting or setting the `Properties.Name` property.